server/controllers/api.py

```diff
@@ -338,18 +367,21 @@ def post(self, user):
         return {'id': score.id, 'backup': 1}
 
 
-class Enrollment(Resource):
+class Enrollment(PublicResource):
```

Comment:
I'm not a huge fan of exposing this info externally. I plan on making this an authed method when I make some more client changes.